prov/rxd/src/rxd_av.c

```diff
@@ -302,14 +296,12 @@ static int rxd_av_lookup(struct fid_av *av, fi_addr_t fi_addr, void *addr,
 			 size_t *addrlen)
 {
 	struct rxd_av *rxd_av;
-	fi_addr_t dg_fiaddr;
+	void *dg_fiaddr;
 
 	rxd_av = container_of(av, struct rxd_av, util_av.av_fid);
-	dg_fiaddr = rxd_av_dg_addr(rxd_av, fi_addr);
-	if (dg_fiaddr == FI_ADDR_UNSPEC)
-		return -FI_ENODATA;
+	dg_fiaddr = ofi_av_get_addr(&(rxd_av->util_av), fi_addr);
```

Comment:
I think we still need the error case here (when the app tries to look up an address that isn't in the av)

Reply:
The ofi_av_init function does not define a default initialization value for the util av  and when the ofi_av_get_addr is called it does not explicitly return FI_ADDR_UNSPEC. so I think the existing check may not work. So may be we need to discuss this a bit more.

Reply:
Ok, yeah maybe just leave as is then